packages/zoe/test/unitTests/contracts/test-auction.js

```diff
@@ -40,7 +40,10 @@ test('zoe - secondPriceAuction w/ 3 bids', async t => {
           Asset: moolaKit.issuer,
           Ask: simoleanKit.issuer,
         });
-        const terms = { timeAuthority: timer, bidDuration: 1n };
+        const terms = {
+          timeAuthority: Promise.resolve(timer),
```

Comment:
I don't understand the purpose of this. Why wrap the `timer` in a promise? I'm equally confused by the other examples below.